speech/system-test/createRecognizer.v2.test.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable */
+
+'use strict';
+
+const {expect} = require('chai');
+const {describe, it} = require('mocha');
+const uuid = require('uuid');
+const sinon = require('sinon');
+const speech = require('@google-cloud/speech').v2;
+
+const {createRecognizerV2} = require('../createRecognizer.v2');
+
+let recognizerId, projectId;
+
+describe('Create a speech recognizer (v2)', () => {
+  const stubConsole = function () {
+    sinon.stub(console, 'error');
+    sinon.stub(console, 'log');
+  }
+
+  const restoreConsole = function () {
+    console.log.restore();
+    console.error.restore();
+  }
+
+  before(async ()=>{
+    const client = new speech.SpeechClient();
+    projectId = await client.getProjectId();
+  })
+
+  beforeEach(stubConsole);
+  afterEach(restoreConsole);
+
+  it('should create a speech recognizer', async () => {
+    recognizerId = `rec-${uuid.v4()}`;
```

Comment:
**Nit:** `recognizerId` doesn't change here, so can we specify it near the top of the file?

e.g. remove the `recognizerId` in line 27 in favor of:
```
const recognizerId = `rec-${uuid.v4()}`;
```